src/pages/EmergencyViewPage/EmergencyViewPage.tsx

```diff
@@ -41,12 +41,11 @@ export const emptyPersonalizedSchedule = {};
 
 export const EmergencyViewPage: React.FC = () => {
   const [selectedTab, updateTab] = useState(0);
-  const venueId = useVenueId() || "";
+  const spaceSlug = useSpaceParams() || "";
```

Comment:
[NOTE]
`useSpaceBySlug()` should probably be able to receive `undefined` (but not missing) argument and deal with it accordingly